src/firefly/js/util/Validate.js

```diff
@@ -100,6 +100,32 @@ export const validateUrl = function(description,valStr) {
     return retval;
 };
 
+export const validateDate = function(description, valStr){
+    var retval = {
+        valid: true,
+        message: ''
+    };
+
+    if(valStr && valStr !== '') {
+        try {
+            var dateObj = new Date(valStr),
+                parsedDate = dateObj.toISOString().substr(0, dateObj.toISOString().indexOf('T'))//Extracting the "date" string from the date object, in yyyy-mm-dd format.
```

Comment:
Don't use commas. This should be two statements.  Add a semicolon. ESlint should catch this. 
Do you have eslint set up in you IntelliJ?

It there are reason why you did not use the validate NPM package here? did it not do the job?

Reply:
- Fixed.
- I don't have the ESlint activated. I will do that.
- The main problem with NPM validate is its not catching entries like for example '02/31/2018'
So, I'm simply throwing the entered date into a JS date obj. and check it after, if the value of the date object is valid or not.

Reply:
Great.  Glad you have a good reason.  Setting up ESlint is important. Any coding standard we do have are defined by how we set up ESlint.